marathon/adapter.go

```diff
@@ -141,3 +140,18 @@ func (m *marathonAdapter) findDependencies(services []*api.Service) map[string]i
 
 	return deps
 }
+
+func (m *marathonAdapter) generateUniqueUID() string {
```

Comment:
The generateUniqueUID() does not need a receiver per say, as it does not use it. We can make it just a helper function.